.travis.yml

```diff
@@ -0,0 +1,14 @@
+language: node_js
+node_js:
+- 5.6
+install:
+- npm install
+script:
+- gulp ci
+deploy:
+  provider: releases
+  api_key:
+    secure: p0DliyK4lDpmS26BGzvpxVh1UtEexDMSMZlokUyJvI+LqBfI8US3pCQDeo8kjAXhMTRJMlqJxm7oll8eyq6VMOt37t8juuBAfNh1WCcJBfojhpbcRPJ7PIhAn8djpJQgPRBYPhBXVU0HAyiryyab6AZdBXxRvEzFPonPwayvWEtJeJdjgBsMdL4TMDd3BkrWZhwCXlIGvRY5pJMb3QiNwLQlxHJ8s8K3emXn4IcPyKbg0jJy6uLTo4kDXFL/MOwB7etYpJKxNlQBcjaa7P8+Zf/xdmhwW3kYgKAO2JGZGF0lF6oQ5EZAbWlCERlOZg4cJmRjM0QovO/L3NXkWj4RzKy+9/5A6vitu3R4GkbRq7sOO7obPXXozSFZHdRzcswsEvI2IlM4KNwRnLQw170FfsgpvaP15nvNVnzIgXP9B/J/rAfyM/4FCPUixs9gEtt4SnhTyoo/Ty6jaCTjPNBvCQbed0Qgb0LjxTfwzTKNl/ScGN1VVp5wieZFrFrqCHj4/GjDEr4ywgApRucqGi7W+0IZO5UbFnG5ksqnwmg+V5tmv3/2AtQytOXqsE7QeMneflM3/L3y3+O18Q30Ocqlbx6R1R+hSjYzZ2vmau3SyI4Lx7cP2VgD2NR68Wz8GdaLyOX1yYEZa7BQ+uuJ3miPPvYc7b+iM0PQthgvlloVd5I=
```

Comment:
The API key or the deployment configuration in general?  The latter is what causes a release to be generated (and ZIP file created) when a tag is attached to a commit.  The former is necessary for Travis to have access to the repo to do the attaching; the key itself is encrypted with a repo-specific key (held by Travis), so it's fine to be in the file.